src/app/tenant/sunbird/MunicipalFinance.html

```diff
@@ -39,8 +39,8 @@
 
 <body>
 <!-- new header design start here  -->
-<div >
-  <div class="nav-header-theme">
+<div class="">
```

Comment:
@shreyas1434shinde  Is blank class needed? If not remove it from all locations.

Reply:
@paritshivani empty class is removed